docs/sr/week01/01-2.md

```diff
@@ -0,0 +1,158 @@
+
+<!--Some of the practical applications of CNNs are powering autonomous driving and analysing medical images.
+-->
+Neke od primena konvolucionih neuronskih mreža su i omogućavanje autonomne vožnje i analiza medicinskih slika.
+
+<!--Although the science and mathematics behind deep learning is fairly understood, there are still some interesting questions that require more research. These questions include: Why do architectures with multiple layers perform better, given that we can approximate any function with two layers? Why do CNNs work well with natural data such as speech, images, and text? How are we able to optimize non-convex functions so well? Why do over-parametrised architectures work?
+-->
+Iako je nauka i matematika iza dubokog učenja dosta dobro shvaćena, idalje postoje zanimljiva pitanja koja treba istražiti. Na primer: Zašto arhitekture sa više slojeva rade bolje, uzevši u obzir da možemo da aproksimiramo funkciju pomoću 2 sloja? Zašto konvolucione neuronske mreže rade dobro sa prirodnim podacima kao što su govor, slike i tekst? Kako uspevamo da toliko dobro optimizujemo nekonveksne funkcije? Zašto arhitekture sa previše parametara rade?
+
+<!--Feature extraction consists of expanding the representational dimension such that the expanded features are more likely to be linearly separable; data points in higher dimensional space are more likely to be linearly separable due to the increase in the number of possible separating planes.
+-->
+Izdvajanje odlika sastoji se od proširivanja dimenzije reprezentacije tako da proširena obeležja verovatnije budu linearno separabilna, tačke u prostoru više dimenzije su verovatnije linearno separabilne zbog povećanja broja potencijalnih separacionih ravni.
+
+<!--Earlier machine learning practitioners relied on high quality, hand crafted, and task specific features to build artificial intelligence models, but with the advent of Deep Learning, the models are able to extract the generic features automatically. Some common approaches used in feature extraction algorithms are highlighted below:
+-->
+Ranije se u primenama mašinskog učenja oslanjalo na kvalitetne, ručno odabrane odlike, specifične za zadatak. Zbog napretka dubokog učenja, modeli su u mogućnosti da automatski izdvoje obeležja. Neki od pristupa korišćenih u izdvajanju odlika:
+
+<!--- Space tiling -->
+<!--- Random Projections-->
+<!--- Polynomial Classifier (feature cross-products)-->
+<!--- Radial basis functions-->
+<!--- Kernel Machines-->
+
+- Popločavanje prostora
+- Nasumične projekcije
+- Polinomijalni klasifikator (unakrsni proizvodi obeležja)
```

Comment:
I find this suggestion more convenient as I never saw anywhere the wording "unakrsni proizvod" and they are synonyms.
```suggestion
- Polinomijalni klasifikator (vektorski proizvodi obeležja)
```